examples/demo/demo.ts

```diff
@@ -19,6 +19,16 @@ import { UnorderedListButton } from '../../packages/plugin-list/src/ListButtons'
 import { IndentButton } from '../../packages/plugin-indent/src/IndentButtons';
 import { OutdentButton } from '../../packages/plugin-indent/src/IndentButtons';
 import { AlignLeftButton, AlignCenterButton, AlignRightButton, AlignJustifyButton } from '../../packages/plugin-align/src/AlignButtons';
+import '!style-loader!css-loader!@fortawesome/fontawesome-free/css/all.css';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-brands-400.woff2';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-brands-400.ttf';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-regular-400.woff2';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-regular-400.ttf';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-solid-900.woff2';
+import '!file-loader?name=./fonts/[name].[ext]!@fortawesome/fontawesome-free/webfonts/fa-solid-900.ttf';
```

Comment:
I tried moving this to `Fontawesome.ts` later and it works. Should I do it in a separate commit?

Reply:
I tried moving these imports to `Fontawesome.ts` which would be cool as then the fonts are loaded automatically when you install the plugin. But... Then for some reason the tests don't work. To me this is a question for... @Goaman ! Is it a plane? Is it a bird? No! It's GOAMAN!

Reply:
Which tests don't work? I tried to remove theses from demo.ts and everything works fine.

Reply:
Did you put them in `Fontawesome.ts`, remove them from `demo.ts` and then run all the tests?

Reply:
@Goaman @dmo-odoo